src/data/markdown/docs/01 guides/04 Results visualization/00 End-of-test Summary.md

```diff
@@ -0,0 +1,198 @@
+
+<CodeGroup labels={[]}>
+
+```
+     ✓ http2 is used
+     ✓ status is 200
+     ✓ content is present
+
+     █ Static Assets
+
+       ✓ status is 200
+       ✓ reused connection
+
+   ✓ check_failure_rate.........: 0.00%   ✓ 0     ✗ 6708
+     checks.....................: 100.00% ✓ 16770 ✗ 0
+     data_received..............: 94 MB   308 kB/s
+     data_sent..................: 1.6 MB  5.2 kB/s
+     group_duration.............: min=134.4ms  avg=177.67ms med=142.75ms p(95)=278.26ms p(99)=353.49ms p(99.99)=983.84ms max=1.01s
+     http_req_blocked...........: min=947ns    avg=1.66ms   med=2.37µs   p(95)=4.65µs   p(99)=38.98µs  p(99.99)=620.34ms max=811.88ms
+     http_req_connecting........: min=0s       avg=536.83µs med=0s       p(95)=0s       p(99)=0s       p(99.99)=208.81ms max=232.16ms
+   ✓ http_req_duration..........: min=131.44ms avg=150.63ms med=138.13ms p(95)=269.81ms p(99)=283.83ms p(99.99)=982.76ms max=1.01s
+     ✗ { staticAsset:yes }......: min=131.44ms avg=153.09ms med=138.2ms  p(95)=271.34ms p(99)=284.22ms p(99.99)=1.01s    max=1.01s
+     http_req_receiving.........: min=33.36µs  avg=2.66ms   med=180.36µs p(95)=2.4ms    p(99)=128.79ms p(99.99)=205.16ms max=205.45ms
+     http_req_sending...........: min=6.09µs   avg=44.92µs  med=35.77µs  p(95)=98.26µs  p(99)=148.49µs p(99.99)=1.09ms   max=5.53ms
+     http_req_tls_handshaking...: min=0s       avg=1.12ms   med=0s       p(95)=0s       p(99)=0s       p(99.99)=447.46ms max=614.35ms
+     http_req_waiting...........: min=131.3ms  avg=147.92ms med=137.57ms p(95)=267.49ms p(99)=282.23ms p(99.99)=982.55ms max=1.01s
+     http_reqs..................: 13416   44.111343/s
+     iteration_duration.........: min=2.28s    avg=3.83s    med=3.82s    p(95)=5.2s     p(99)=5.36s    p(99.99)=6.1s     max=6.18s
+     iterations.................: 3354    11.027836/s
+     vus........................: 1       min=1   max=50
+     vus_max....................: 50      min=50  max=50
+```
+
+</CodeGroup>
+
+A few options can affect how this report behaves:
+- The [`--summary-trend-stats` option](/using-k6/options#summary-trend-stats) allows you to define which stats for [Trend metrics](/javascript-api/k6-metrics/trend) will be calculated and shown.
+- The [`--summary-time-unit` option](/using-k6/options#summary-time-unit) forces k6 to use a fixed time unit for _all_ time values in the summary.
+- [`no-summary`](/using-k6/options#no-summary) completely disables the report generation. Since k6 v0.30.0 that includes `--summary-export` and `handleSummary()` (see below).
+
+## Summary export to a JSON file
+
+Since k6 v0.26.0, k6 has had the [`--summary-export=path/to/file.json` option](/using-k6/options#summary-export) for local test runs. It exports some of the summary report data to a JSON file format.
```

Comment:
```suggestion
Since version 0.26.0 k6 has had the [`--summary-export=path/to/file.json` option](/using-k6/options#summary-export) for local test runs. It exports some of the summary report data to a JSON file format.
```